src/github.com/getlantern/flashlight/analytics/analytics.go

```diff
@@ -0,0 +1,68 @@
+package analytics
+
+import (
+	"net/http"
+
+	"github.com/mitchellh/mapstructure"
+
+	"github.com/getlantern/analytics"
+	"github.com/getlantern/flashlight/ui"
+	"github.com/getlantern/golog"
+)
+
+const (
+	messageType = `Analytics`
+)
+
+var (
+	log        = golog.LoggerFor("flashlight.analytics")
+	service    *ui.Service
+	httpClient *http.Client
+)
+
+func Configure(autoReport *bool, newClient *http.Client) {
+	httpClient = newClient
+
+	if autoReport != nil && *autoReport {
+		err := Start()
+		if err != nil {
+			log.Errorf("Error starting analytics service: %q", err)
+		}
+	}
```

Comment:
Right, but why do we need to keep the service running too? Why have an unused service present at all? Wouldn't it be better to close the connection and free up resources?